Objects/listobject.c

```diff
@@ -2728,6 +2728,35 @@ list___init___impl(PyListObject *self, PyObject *iterable)
     return 0;
 }
 
+static PyObject *
+list_vectorcall(
+    PyObject *type, PyObject * const*args,
+    size_t nargsf, PyObject *kwnames)
+{
+    if (kwnames && PyTuple_GET_SIZE(kwnames) != 0) {
+        PyErr_Format(PyExc_TypeError, "list() takes no keyword arguments");
+        return NULL;
+    }
+    size_t nargs = PyVectorcall_NARGS(nargsf);
+    if (nargs > 1) {
+        PyErr_Format(PyExc_TypeError, "list() expected at most one argument, got %zd", nargs);
```

Comment:
can "one" be changed to 1 for `list()` and `dict()` to be same as previous error messages?